src/trove/lists.scrbl

```diff
@@ -1179,54 +1169,9 @@
       (a-arrow
         (a-app (a-id "List" (xref "lists" "List")) "a")
         (a-app (a-id "List" (xref "lists" "List")) "a"))))
-  (fun-spec
-    (name "get-help")
-    (arity 2)
-    (params [list: leaf("a")])
-    (args ("lst" "n"))
-    (return "a")
-    (contract
-      (a-arrow
-        (a-app (a-id "List" (xref "lists" "List")) "a")
-        (a-id "Number" (xref "<global>" "Number"))
-        "a"))
-    (doc
-      "Returns the nth element of the given list, or raises an error if n is out of range"))
-  (fun-spec
-    (name "set-help")
-    (arity 3)
-    (params [list: leaf("a")])
-    (args ("lst" "n" "v"))
-    (return "a")
-    (contract
-      (a-arrow
-        (a-app (a-id "List" (xref "lists" "List")) "a")
-        (a-id "Number" (xref "<global>" "Number"))
-        "Any"
-        "a"))
-    (doc
-      "Returns a new list with the same values as the given list but with the nth element\n        set to the given value, or raises an error if n is out of range"))
   (fun-spec
     (name "length")
     (arity 1))
-  (fun-spec
```

Comment:
These bindings, on the other hand -- I'm not sure how they got here.  Did they move from lists.arr to another file?  Anyway it's good you've removed them :)